frontend/src/app/(pages)/assignments/submissions/[id]/page.tsx

```diff
@@ -0,0 +1,222 @@
+    },
+    {
+      id: "grades",
+      label: "Grades",
+      content:
+        "Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
+    },
+  ];
+
+  return (
+    <div>
+      {assignment && (
+        <div className="h-screen">
+          <h1>Submissions</h1>
+          <div className="grid grid-cols-2 gap-4">
+            <div className="col-span-1">
+              <div className="flex gap-2">
+                <div>
+                  <h1 className="text-3xl font-semibold ">
+                    {assignment.title}
+                  </h1>
+                  <div className="flex flex-col ml-4 my-4 gap-2">
+                    <p className="text-lg font-semibold">
+                      Due on:{" "}
+                      <span className="italic font-medium">
+                        {DateUtils.parseTimestampToDate(assignment.deadline)}
+                      </span>
+                    </p>
+                    <p className="text-lg font-semibold">
+                      Number of questions:{" "}
+                      <span className="italic font-medium">
+                        {assignment.numberOfQuestions}
+                      </span>
+                    </p>
+                  </div>
+                </div>
+              </div>
+              <div className="flex px-0 py-4 mb-6">
+                <div className="w-full px-5">
+                  {/* Question title */}
+                  <div className="flex space-x-4">
+                    <div className="flex-1 mr-2 text-xl font-semibold">
+                      {/* {question.title} */}
+                    </div>
+                  </div>
+
+                  <Divider className="mt-4 mb-2" />
+
+                  {/* Question description */}
+                  <div className="flex mt-3">
+                    <div className="text-md text-justify">
+                      {/* {parse(question.description)} */}
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div className="col-span-1">
+              <div className="row-span-1 border border-black">
+                <Editor
+                  options={{
+                    minimap: {
+                      enabled: false,
+                    },
+                    readOnly: true,
+                  }}
+                  height="50vh"
+                  // theme="vs-dark"
+                  language="python"
+                  // defaultValue={CODE_SNIPPETS[language]}
+                  onMount={onMount}
+                  value={code}
+                  // onChange={(value) => setValue(value)}
+                />
+              </div>
+              <Spacer y={4} />
+              <div className="row-span-1">
+                <div className="flex w-full flex-col">
+                  <Tabs aria-label="Dynamic tabs" items={tabs}>
```

Comment:
Perhaps you can change the background color when a tab is on focus, as the default one seems to be too dark:
![image](https://github.com/tryyang2001/CS3213-Frontend-Management-System/assets/69421979/4a7b46cb-9156-498d-9caa-24669c6fe4dc)

Reply:
For now I changed the highlighted colour to blue:
<img width="511" alt="Screenshot 2024-03-18 at 1 44 42 AM" src="https://github.com/tryyang2001/CS3213-Frontend-Management-System/assets/100022021/8e015844-d2a5-4ac3-82e3-bcfce00f1f42">